spec/system/admin_split_assignments_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'bulk assign flow' do
```

Comment:
This description might be copied? Should it be more like "view assignments page"

Reply:
Thank you! Updated